app/models/article.rb

```diff
@@ -4,6 +4,19 @@ class Article < ActiveRecord::Base
   validates :title, presence: true, length: { minimum: 5 }
 
   after_commit :_flush_all_cache
+  has_many :comments
+
+  def self.cached_comments(id)
```

Comment:
@phendych why static method?
this should be an instance method, then you'd already have the object id, you can just do
self.comments instead of find(id).comments